test/custom_assertions.hpp

```diff
@@ -0,0 +1,83 @@
+#ifndef CUSTOM_ASSERTIONS_HPP_
```

Comment:
If you put this file here, there might be problems with "duplicate definition" errors once the psim pointer on FlightSoftware is updated. We'll see if it happens and if it does, we can hot-fix the issue

Reply:
It should be totally fine as long as we only use this in tests themselves.